CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 - Added PKTINFO cmsg support on Android/iOS/Linux/MacOS.
 - Added V6PKTINFO cmsg support on Android/FreeBSD/iOS/Linux/MacOS.
-- ([#990](https://github.com/nix-rust/nix/pull/990))
+  ([#990](https://github.com/nix-rust/nix/pull/990))
```

Comment:
This still isn't correct because the link is only associated with the V6PKTINFO bullet.  You need to either duplicate the link for both bullets, or collapse everything into a single bullet.